frontend/src/ui/Video.tsx

```diff
@@ -101,27 +107,29 @@ type ThumbnailReplacementProps = {
     audioOnly: boolean;
     isDark: boolean;
     isUpcoming?: boolean;
+    deletionIsPending?: boolean;
 }
 export const ThumbnailReplacement: React.FC<ThumbnailReplacementProps> = (
-    { audioOnly, isDark, isUpcoming }
+    { audioOnly, isDark, isUpcoming, deletionIsPending }
 ) => {
     // We have no thumbnail. If the resolution is `null` as well, we are
     // dealing with an audio-only event and show an appropriate icon.
     // Otherwise we use a generic icon.
-    const icon = audioOnly ? <LuVolume2 /> : <LuFilm />;
+    const icon = deletionIsPending ? <LuTrash /> : audioOnly ? <LuVolume2 /> : <LuFilm />;
```

Comment:
Nested ternaries are confusing to read, especially when they are not properly formatted or surrounded by braces.